src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -995,3 +995,17 @@ def test_ffpyplayer(pyi_builder):
     pyi_builder.test_source("""
         import ffpyplayer.player
         """)
+
+
+@importorskip("twisted")
+def test_ffpyplayer(pyi_builder):
```

Comment:
The test name is probably a copy&paste typo.

Also, I think it would make sense to split this into two tests (e.g., `test_twisted_default_reactor` and `test_twisted_asyncio_reactor`) so each case is tested independently of the other.

Reply:
I've fixed both of those.